front/lib/models/assistant/actions/tables_query.ts

```diff
@@ -223,6 +227,10 @@ AgentTablesQueryAction.init(
       type: DataTypes.JSONB,
       allowNull: true,
     },
+    resultsFileSnippet: {
```

Comment:
ah no, here :)

Reply:
🤔 IIRC our discussion I thought we mentioned doing this on the `files` table, did we change opnion?

Reply:
Nope I think we misunderstood each other because I never planned to do this. Storing data on the File object in front sounds a bit strange to me and this is quite specific to tables query. 
Also, this saves be a bunch of trouble having to pull the file resource in various places